zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/auth/MiniKdc.java

```diff
@@ -103,7 +103,7 @@ public static void main(String[] args) throws Exception {
         Properties userConf = new Properties();
         InputStreamReader r = null;
         try {
-            r = new InputStreamReader(new FileInputStream(file), Charsets.UTF_8);
+            r = new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8);
```

Comment:
Why is this change?

Reply:
I can't remember. I have to check

Reply:
@anmolnar 
it is because org.apache.commons.io.Charsets.UTF_8 is deprecated